src/ray/rpc/grpc_client.h

```diff
@@ -88,6 +88,51 @@ inline std::shared_ptr<grpc::Channel> BuildChannel(
   return channel;
 }
 
+inline grpc::ChannelArguments CreateClientDefaultChannelArguments() {
+  // Please refer to explanation of gRPC returned statuses for more details:
+  // REF: https://grpc.github.io/grpc/core/md_doc_statuscodes.html
+  std::string service_config_json = R"(
+  {
+    "methodConfig": [{
```

Comment:
Can you explain how/why these methods?

Reply:
Yeah, i couldn't leave the comments in raw literals, but basically what i've done is followng:
 
1. Allowing to retry any read-only APIs
2. Mutating APIs i did a quick scan and if it's implemented idempotently, i've added it as well